src/ragger/backend/interface.py

```diff
@@ -66,44 +59,41 @@ def last_async_response(self) -> Optional[RAPDU]:
         """
         return self._last_async_response
 
-    @property
-    def raises(self) -> bool:
+    @abstractmethod
+    def __enter__(self) -> "BackendInterface":
+        raise NotImplementedError
+
+    @abstractmethod
+    def __exit__(self, exc_type: Optional[Type[BaseException]], exc_val: Optional[BaseException],
+                 exc_tb: Optional[TracebackType]):
+        raise NotImplementedError
+
+    def get_raise_policy(self) -> RaisePolicy:
```

Comment:
Getter and setter in Python are usually done through properties:
```python
@property 
def raise_policy(self): ...
@property.setter
def raise_policy(self, value): ...
```
And then used as a plain field:
``` python
backend.raise_policy = RaisePolicy.RAISE_NOTHING
```

In this case, as nothing is checked or processed, I'm not sure adding an abstraction is worth it. (?)

Reply:
I removed the abstraction and made raise_policy a plain attribute